src/background/controllers/inpageAccountController.ts

```diff
@@ -0,0 +1,42 @@
+import QryptoController from '.';
+import IController from './iController';
+import { MESSAGE_TYPE } from '../../constants';
+
+export default class InpageAccountController extends IController {
+
+  constructor(main: QryptoController) {
+    super('inpageAccount', main);
+    chrome.runtime.onMessage.addListener(this.handleMessage);
+    this.initFinished();
+  }
+
+  public sendQryptoAccountValuesToActiveTab = () => {
+    chrome.tabs.query({ active: true, currentWindow: true }, ([{ id: tabID }]) => {
```

Comment:
hmm my guess is that it is not a problem for the qryptoAccountValues because it is all local fetches and I don't think the user can change tabs that fast, but maybe for the rpc calls.
I will add something for them both to use for consistency though.

Reply:
As discussed, lets add registration logic so the dapp can register their `tabID`.